index.html

```diff
@@ -4,76 +4,183 @@
+                        <img src="assets/liza-pooor-QHutOO4jiRw-unsplash.jpg" class="d-block w-100" alt="...">
+                        <div class="carousel-caption d-none d-md-block">
+                            <h5>First slide label</h5>
+                            <p>Projet 1</p>
+                        </div>
+                        <br>
                     </div>
-                </div>
-            </div>
-            <div class="col-4">
-                <div class="card">
-                    <img src="./assets/man.jpg" class="card-img-top" alt="...">
-                    <div class="card-body">
-                        <h5 class="card-title">Card title</h5>
-                        <p class="card-text">Some quick example text to build on the card title and make up the bulk of
-                            the card's content.</p>
-                        <a href="#" class="btn btn-primary">Go somewhere</a>
+                    <div class="carousel-item" data-bs-interval="2000">
+                        <img src="assets/cdc-k0KRNtqcjfw-unsplash.jpg" class="d-block w-100" alt="...">
+                        <div class="carousel-caption d-none d-md-block">
+                            <h5>Second slide label</h5>
+                            <p>Projet 2</p>
+                        </div>
+                        <br>
+                    </div>
+                    <div class="carousel-item">
+                        <img src="assets/stefan-schauberger-Tper6bHeSUo-unsplash.jpg" class="d-block w-100" alt="...">
+                        <div class="carousel-caption d-none d-md-block">
+                            <h5>Third slide label</h5>
+                            <p>Projet 3</p>
+                        </div>
+                        <br>
                     </div>
+                    
                 </div>
+                <button class="carousel-control-prev" type="button" data-bs-target="#carouselExampleDark"
+                    data-bs-slide="prev">
+                    <span class="carousel-control-prev-icon" aria-hidden="true"></span>
+                    <span class="visually-hidden">Previous</span>
+                </button>
+                <button class="carousel-control-next" type="button" data-bs-target="#carouselExampleDark"
+                    data-bs-slide="next">
+                    <span class="carousel-control-next-icon" aria-hidden="true"></span>
+                    <span class="visually-hidden">Next</span>
+                </button>
             </div>
         </div>
     </div>
+
+
+    <div class="row">
+        <hr class="ligne">
+    </div>
+
+
+
+
 </body>
-<script src="js/bootstrap.min.js"></script>
+
+
+<footer>
+
+    <p class="text-center">Suivez moi sur mes réseaux</p>
+    <div class="container text-center">
+        <div class="row">
+            
+            <div class="col">
+                <a href="https://www.facebook.com/daniel.lelievre.39/"><img src="assets/Facebook.png" alt=""></a>
```

Comment:
Try to user `target='_blank'` property for your external links. This will make the page opens in a new tab instead to replace your current page.

Reply:
Give some alt description for screen readers. This will increase your SEO and also be in the acessibility standards.